scripts/validate-ci

```diff
@@ -25,6 +25,3 @@ fi
 
 echo Checking if released versions are not changed
 go run ./pkg/validation/validation.go release-v2.8
-
-echo Checking the generated regsync.yaml file
```

Comment:
Approved, but where do we do this instead? Or do we not need to?

Reply:
We decided it was nice to have but not required on pull request, since it was fetching rancher prime registry on PR CI. The check didn't actually fail, just listed all the new tags that needed to be synced which most of us didn't check anyway.